src/data-loaders/HttpDataLoader.js

```diff
@@ -16,7 +16,11 @@ export default class HttpDataLoader {
         data: req.params.body,
       });
 
-      return data;
+      return {
+        headers,
+        statusCode: status,
+        body: JSON.stringify(data)
+      }
```

Comment:
@stschwark Well spotted! 
I haven't played with this enough to catch this.

Thank you!